docs/js/doc-page.js

```diff
@@ -21,21 +21,90 @@ function removeDoubleSlashes(str) {
   return str.replace(/\/{2,}/g, '/');
 }
 
-export default ({ data, location }) => {
-  const page = data.mdx || {};
-  const title =
-    page.headings && page.headings[0] ? page.headings[0].value : 'Docs';
+function getPathFromSlug(str) {
+  return removeDoubleSlashes(`/${str}/`);
+}
 
-  const aesPage = isAesPage(page.fields.slug);
-  const apiGatewayPage = isApiGatewayPage(page.fields.slug);
+function relativeToAbsUrl(slug) {
+  return 'https://www.getambassador.io' + getPathFromSlug(slug);
+}
 
-  const metaDescription = page.frontmatter
-    ? page.frontmatter.description
-    : null;
+function getMainDocsUrl(slug) {
+  return relativeToAbsUrl(slug.match(/\/docs\/[\d\.(latest)]*/)[0]);
+}
```

Comment:
I guess this is the part that's erroring? It will throw a type error if the slug doesn't match that regex.

Reply:
That regex doesn't make sense to me.

`return relativeToAbsUrl(getPathFromSlug(slug).match(/\/docs\/[^\/]+/)[0])` perhaps?

Reply:
There, I've made it clearer, commented it and fixed the build errors :D